[new product] Mondoo cnspec (#6471)
captn3m0/endoflife.date: products/cnspec.md

---
title: cnspec
category: app
tags: mondoo
permalink: /cnspec
versionCommand: cnspec version
releasePolicyLink: https://mondoo.com/docs/cnspec/cnspec-adv-install/update/

> changelogTemplate: "https://mondoo.com/docs/releases/mondoo-__LATEST__-is-out/"

Reviewer: ```suggestion
changelogTemplate: https://github.com/mondoohq/cnspec/releases/tag/v__LATEST__
```

The current `changelogTemplate` generate wrong links such as https://mondoo.com/docs/releases/mondoo-11.35.0-is-out/.

https://mondoo.com/docs/releases works only for minor releases whereas the link is generated for the full version, so it would be best to link to GitHub releases instead.

eoasColumn: true
releaseDateColumn: true

auto:
  methods:
  -   github_releases: mondoohq/cnspec

identifiers:
-   cpe: cpe:2.3:a:cnspec:cnspec

# eol(x) = releaseDate(x+2)

> # eoas(x) = releaseDate(x+1)

Reviewer: ```suggestion
# eoas(x) = releaseDate(x+1)
# eol(x) = releaseDate(x+2)
```

Just changed the order ;)

releases:
-   releaseCycle: "11"
    releaseDate: 2024-04-17
    eoas: false
    eol: false
    latest: "11.35.0"
    latestReleaseDate: 2024-12-17

-   releaseCycle: "10"
    releaseDate: 2024-01-23
    eoas: 2024-04-17
    eol: false
    latest: "10.12.2"
    latestReleaseDate: 2024-04-16

-   releaseCycle: "9"
    releaseDate: 2023-10-05
    eoas: 2024-01-23
    eol: 2024-04-17
    latest: "9.14.0"
    latestReleaseDate: 2024-01-09

-   releaseCycle: "8"
    releaseDate: 2023-03-09
    eoas: 2023-10-05
    eol: 2024-01-23
    latest: "8.28.4"
    latestReleaseDate: 2023-09-19

-   releaseCycle: "7"
    releaseDate: 2022-10-18
    eoas: 2023-03-09
    eol: 2023-10-05
    latest: "7.19.2"
    latestReleaseDate: 2023-03-02

-   releaseCycle: "6"
    releaseDate: 2022-10-11
    eoas: 2022-10-18
    eol: 2023-03-09
    latest: "6.19.0"
    latestReleaseDate: 2022-10-11

---

> > [cnspec](https://mondoo.com/cnspec/) cnspec is an open source, cloud-native tool that assesses

Reviewer 2: ```suggestion
> [cnspec](https://mondoo.com/cnspec/) is an open source, cloud-native tool that assesses
```

> > [cnspec](https://mondoo.com/cnspec/) is an open source, cloud-native tool that assesses
> > the security of your entire infrastructure.

Reviewer: ```suggestion
> [cnspec](https://mondoo.com/cnspec/) is an open source, cloud-native security and policy tool.
> It assesses an entire infrastructure's security and compliance and finds vulnerabilities and
> misconfigurations across public and private cloud environments, Kubernetes clusters, containers,
> container registries, servers, endpoints, SaaS products, infrastructure as code, APIs, and more.
```

Inspired by https://github.com/mondoohq/cnspec?tab=readme-ov-file#cnspec, WDYT ?

> 

Reviewer: Is it the same level of support between versions N and N-1 ?

Author: Same level for both

> cnspec follows an N-1 support strategy (meanings that the 2 most recent major versions receive

Reviewer 2: ```suggestion
cnspec follows an N-1 support strategy (meaning that the two most recent major versions receive
```

> cnspec follows an N-1 support strategy (meaning that the two most recent major versions receive
> security and bug fixes).

Reviewer: ```suggestion
cnspec follows an N-1 support strategy, meaning the current major version (N) and the immediate previous major version (N-1) are officially supported with non-breaking changes, bug and security fixes. During this time, both major versions can communicate with the Mondoo Platform and execute all published policies and query packs.
```

Inspired by https://mondoo.com/docs/cnspec/cnspec-adv-install/update/#supported-releases. I don't know cnspec, but the last sentence (`During this time, both major...`) seems also important, WDYT ?

> security and bug fixes)

Reviewer 2: ```suggestion
security and bug fixes).
```
